projects/igniteui-angular/src/lib/grids/filtering/grid-filtering-row.component.ts

```diff
@@ -77,9 +77,13 @@ export class IgxGridFilteringRowComponent implements AfterViewInit {
 
     set column(val) {
         if (val) {
+            const initial = this._column === null;
             this._column = val;
 
             this.expressionsList = this.filteringService.getExpressions(this._column.field);
+            if (!initial) {
+                this.expressionsList.forEach(ex => ex.isSelected = false);
```

Comment:
Instead of setting isSelected to false here, it is better to set it in the commitInput() method.
```
    public commitInput() {
        // this.chipsArea.chipsList.filter(chip => chip.selected = false);
        this.expressionsList.forEach(item => item.isSelected = false);
```

Reply:
That is braking chip commit with mouse click.